### PR TITLE
Define MSTest to support managed sources only

### PR DESCRIPTION
**Pull request by mayankbansal018**
Following RFC: https://github.com/Microsoft/vstest-docs/blob/master/RFCs/0020-Improving-Logic-To-Pass-Sources-To-Adapters.md

### CODE REVIEW

**Comment by mayankbansal018** [comment]
@dotnet-bot test this please

**Comment by jayaranigarg** [comment]
Make sure you validate end to end flow before pushing this in.

File: src/Adapter/MSTest.CoreAdapter/MSTestDiscoverer.cs [hunk 1]
```diff
@@ -20,6 +21,7 @@ namespace Microsoft.VisualStudio.TestPlatform.MSTest.TestAdapter
     [FileExtension(".appx")]
     [FileExtension(".dll")]
     [FileExtension(".exe")]
+    [Category("managed")]
```
Comment (smadala) [inline src/Adapter/MSTest.CoreAdapter/MSTestDiscoverer.cs:24]: A comment will be helpful.

**Comment by mayankbansal018** [comment]
@dotnet-bot  test this please

**Comment by mayankbansal018** [comment]
@dotnet-bot test this please

**Comment by mayankbansal018** [comment]
@dotnet-bot test this please

**Comment by mayankbansal018** [comment]
@dotnet-bot test this please

File: netci.groovy [hunk 2]
```diff
@@ -28,7 +28,8 @@ def branch = GithubBranchName
             }
         }
 
-        Utilities.setMachineAffinity(newJob, 'Windows_NT', 'latest-or-auto-dev15-0')       
+         // Move to latest VS(15.6) machines
+        Utilities.setMachineAffinity(newJob, 'Windows_NT', 'latest-or-auto-dev15-0')
```
Comment (jayaranigarg) [inline netci.groovy:32]: This is same string only. We didn't update it to point to 15.6 machines?

**Pull request closed without merging**